pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/DSMServer.java

```diff
@@ -588,7 +588,7 @@ protected void setupCustomRouting(@NonNull Config cfg) {
         get(apiRoot + RoutePath.CLINICAL_KIT_ENDPOINT, new ClinicalKitsRoute(notificationUtil), new JsonTransformer());
 
         if (!cfg.getBoolean("ui.production")) {
-            NonPepperKitCreationService nonPepperKitCreationService = new NonPepperKitCreationService();
+            NonPepperKitCreationService nonPepperKitCreationService = new NonPepperKitCreationService(new NonPepperStatusKitService());
             post(apiRoot + RoutePath.SHIP_KIT_ENDPOINT, new JuniperShipKitRoute(nonPepperKitCreationService), new JsonTransformer());
 
             NonPepperStatusKitService nonPepperStatusKitService = new NonPepperStatusKitService();
```

Comment:
It looks like there are two instances of the NonPepperStatusKitService? Is that intentional? Is so, could service creation be done in the Route code (here and above)?